monster.py

```diff
@@ -0,0 +1,76 @@
+import random
+from hero import Hero
+
+
+class Monster:
+    def __init__(self, name_monster: str, hero_level) -> None:
+        self._name_monster = name_monster
+        if hero_level == 1:
+            self._level = random.randint(hero_level, hero_level + 1)
+        else:
+            self._level = random.randint(hero_level - 1, hero_level + 1)
+        self._damage = 1.2 * self.level
+        self._hp = 5 * self.level
+        self._hp_max = 6 * self.level
+
+    life_percent = 30
+    level_percent = 20
+    k = 5
```

Comment:
this three fields are not doing anything in this class, and you dont have class veriables that belong to Monster that should be used